CircleciScripts/maven-release.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+export PATH=$JAVA_HOME/bin:$PATH
+export PATH=$ANDROID_HOME/tools:$PATH
+export PATH=$ANDROID_HOME/platform-tools:$PATH
+export PATH=$ANDROID_HOME:$PATH
+
+
+
+pom="pom.xml"
+settings="CircleciScripts/mvhome/settings.xml"
+localrepo="${HOME}/.m2" 
+skipRemoteStaging=${SkipRemoteStaging}
+autoReleaseAfterClose=${AutoReleaseAfterClose}
+gpg_homedir="${HOME}/code/gpghome"
+sonatype_username=${SonatypeUsername}
+sonatype_password=${SonatypePassword}
+gpg_keyname=${GpgKeyname}
+gpg_passphrase="${GpgPassphrase}"
+echo "gpg_passphrase:$gpg_passphrase"
+
+# Perm gen space is increased for maven command
+MAVEN_OPTS="-Xms1024m -Xmx2048m -XX:PermSize=512m -XX:MaxPermSize=1024m"
+
+mvn clean deploy -X -e -Ppublishing \
```

Comment:
where are we getting credentials for maven deploy?

Reply:
from project environment variable